pkg/docker/docker.go

```diff
@@ -544,8 +545,11 @@ func runContainerTar(opts RunContainerOptions, imageMetadata *docker.Image) (cmd
 	return cmd, tarDestination
 }
 
-// this funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
-func runContainerDockerRun(container *docker.Container, d *stiDocker, image string) {
+// dumpContainerInfoAndWait dumps information about a running container (port/ip/etc) and then
+// waits for the container to exit (which it may never do).  Used when --run=true is passed
+// because the user wants to run the newly built image immediately.
+// This function runs forever, ensure a sig handler is installed elsewhere to handle cntrl-c.
```

Comment:
That's why I had the question mark. I normally see it as `ctrl-`, or `c-`
or `^` and all sorts of capitalization. Wikipedia does list a bunch of ways
different platforms call it, I personally never came across cntrl, first
time today :-)

https://en.wikipedia.org/wiki/Control_key

Rodolfo Carvalho

OpenShift Developer Experience

On Fri, Apr 1, 2016 at 10:26 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/docker/docker.go
> <https://github.com/openshift/source-to-image/pull/452#discussion_r58262724>
> :
>
> > @@ -544,8 +545,11 @@ func runContainerTar(opts RunContainerOptions, imageMetadata *docker.Image) (cmd
> >  	return cmd, tarDestination
> >  }
> >
> > -// this funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
> > -func runContainerDockerRun(container *docker.Container, d *stiDocker, image string) {
> > +// dumpContainerInfoAndWait dumps information about a running container (port/ip/etc) and then
> > +// waits for the container to exit (which it may never do).  Used when --run=true is passed
> > +// because the user wants to run the newly built image immediately.
> > +// This function runs forever, ensure a sig handler is installed elsewhere to handle cntrl-c.
>
> no, i'm not aware of any particular era/cultural shift on that one,
> probably just a personal affectation on my part. my comment was more "cntrl
> isn't explicitly wrong (ie it's not a typo like, say, crtl or something),
> and it's a comment on a private function in a utility package, not public
> documentation"
>
> anyway i've changed it in the latest push.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/source-to-image/pull/452/files/64acc1bf3a18541084b4fb1c3ae969e419466e6d#r58262724>
>